<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabPosition">
         <enum>QTabWidget::West</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="usesScrollButtons">
         <bool>true</bool>
        </property>
        <property name="documentMode">
         <bool>false</bool>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="widget">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_employee_task">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QPushButton" name="createTaskButton">
            <property name="text">
             <string>Добавить задачу</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="deleteTaskButton">
            <property name="text">
             <string>Удалить задачу</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="getLastTaskButton">
            <property name="text">
             <string>Получить список просроченных задач</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="getTaskTable">
            <property name="text">
             <string>Получить список задач</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="4">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="txtTaskTable">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="text">
               <string>Текущий список задач</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="tableViewEmployeeTask"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_create_task">
         <attribute name="title">
          <string>Страница</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <widget class="QLabel" name="txrSurname">
                <property name="text">
                 <string>Фамилия</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="surnameEdit"/>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="txtName">
                <property name="text">
                 <string>Имя: </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="nameEdit"/>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="txtPantronymic">
                <property name="text">
                 <string>Отчество</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="pantronymicEdit"/>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="txtTask">
                <property name="text">
                 <string>Наименование задачи</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="TaskEdit"/>
              </item>
             </layout>
            </item>
            <item row="4" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="txtDate">
                <property name="text">
                 <string>Дата выполнения</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dateEdit"/>
              </item>
             </layout>
            </item>
            <item row="5" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QCheckBox" name="Dependency">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>Зависимость от других задач</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayoutChoseTask">
                <item>
                 <widget class="QCheckBox" name="isParenrTask">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>Родительская задача</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="isTabTask">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>Подзадача</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayoutParentTask">
                <item>
                 <widget class="QLabel" name="txtParentTask">
                  <property name="text">
                   <string>Наименование родительской задачи:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="ParentTaskEdit"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="6" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="txtStatus">
                <property name="text">
                 <string>Cтатус: </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="statusEdit"/>
              </item>
             </layout>
            </item>
            <item row="7" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QPushButton" name="createTaskButton_2">
                <property name="text">
                 <string>Добавить задачу</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="returnToTableButton">
                <property name="text">
                 <string>Вернуться к списку задач</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_delete_task">
         <attribute name="title">
          <string>Страница</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="txtSurnameD">
              <property name="text">
               <string>Фамилия</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="surnameEditD"/>
            </item>
           </layout>
          </item>
          <item row="1" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="txtNameD">
              <property name="text">
               <string>Имя: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="nameEditD"/>
            </item>
           </layout>
          </item>
          <item row="2" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="txtPantronymicD">
              <property name="text">
               <string>Отчество</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pantronymicEditD"/>
            </item>
           </layout>
          </item>
          <item row="3" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="txtTaskNameD">
              <property name="text">
               <string>Наименование задачи</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="taskNameEditD"/>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="deleteTaskButton_2">
            <property name="text">
             <string>Удалить задачу</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="returnToTableButtonD">
            <property name="text">
             <string>Вернуться к списку задач</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
